### PR TITLE
Update rhacs RBAC resource list

### PR DESCRIPTION
**Pull request by rhybrillou**
Version(s): 4.0 +

Issue:

Link to docs preview:
https://58080--docspreview.netlify.app/openshift-acs/latest/operating/manage-user-access/manage-role-based-access-control-3630.html#rbac-system-roles-3630_manage-role-based-access-control

QE review:
- [ ] QE has approved this change.

Additional information:

Adjust the list of available permissions in RHACS role-based access control

Merge into `rhacs-docs` and cherrypick into `rhacs-docs-4.0`

### CODE REVIEW

File: modules/rbac-resource-definitions.adoc [hunk 1]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
-| Create, modify, or delete scheduled compliance runs.
+| Deployment
+| View deployments (workloads) in secured clusters.
+| N/A
 
-| ComplianceRuns
-| View recent compliance runs and their completion status.
-| Trigger compliance runs.
+| DeploymentExtension
+| View
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:77]:
```suggestion
a| View the following items:
```

File: modules/rbac-resource-definitions.adoc [hunk 2]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
 
-| Config
-| View options for data retention, security notices, and other related configurations.
-| Modify these configurations.
-//TODO: Add link to security notices and data retention
+* process baselines.
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:79]:
```suggestion
* Process baselines
```

File: modules/rbac-resource-definitions.adoc [hunk 3]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
 
-| DebugLogs
-| View the current logging verbosity level in {product-title} components.
-| Modify the logging level.
+* process activity in deployments.
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:81]:
```suggestion
* Process activity in deployments
```

File: modules/rbac-resource-definitions.adoc [hunk 4]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
 
-| Deployment
-| View deployments (workloads) in secured clusters.
-| N/A
+* risk results.
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:83]:
```suggestion
* Risk results
```

File: modules/rbac-resource-definitions.adoc [hunk 5]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
+| Modify
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:84]:
```suggestion
a| Modify the following items:
```

File: modules/rbac-resource-definitions.adoc [hunk 6]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
+
+* process baselines (add or remove processes).
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:86]:
```suggestion
* Process baselines (add or remove processes)
```

File: modules/rbac-resource-definitions.adoc [hunk 7]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
 | Image
 | View images, their components, and their vulnerabilities.
 | N/A
 //TODO: Add link to VIew images topic
 
-| ImageComponent
-| _Internal use only_
-| _Internal use only_
+| Integration
+| View:
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:101]:
```suggestion
a| View the following items:
```

File: modules/rbac-resource-definitions.adoc [hunk 8]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
 
-| ImageIntegration
-| List existing image registry integrations.
-| Create, edit, or delete image registry integrations.
-//TODO: Add link to image registry integrations
+* existing API tokens.
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:103]:
```suggestion
* Existing API tokens
```

File: modules/rbac-resource-definitions.adoc [hunk 9]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
 
-| ImbuedLogs
-| _Internal use only_
-| _Internal use only_
+* existing integrations with automated backup systesm such as AWS S3.
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:105]:
```suggestion
* Existing integrations with automated backup systems such as Amazon Web Services (AWS) S3
```

File: modules/rbac-resource-definitions.adoc [hunk 10]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
+
+* existing image registry integrations.
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:107]:
```suggestion
* Existing image registry integrations
```

File: modules/rbac-resource-definitions.adoc [hunk 11]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
+
+* existing integrations for notification systems like email, Jira or webhooks.
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:109]:
```suggestion
* Existing integrations for notification systems like email, Jira, or webhooks
```

File: modules/rbac-resource-definitions.adoc [hunk 12]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
+| Modify
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:110]:
```suggestion
a| Modify the following items:
```

File: modules/rbac-resource-definitions.adoc [hunk 13]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
+
+* API tokens (create new or revoke existing ones).
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:112]:
```suggestion
* API tokens (create new tokens or revoke existing tokens)
```

File: modules/rbac-resource-definitions.adoc [hunk 14]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
+
+* the configurations of backup integrations.
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:114]:
```suggestion
* The configurations of backup integrations
```

File: modules/rbac-resource-definitions.adoc [hunk 15]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
+
+* image registry integrations (create/edit/delete).
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:116]:
```suggestion
* Image registry integrations (create/edit/delete)
```

File: modules/rbac-resource-definitions.adoc [hunk 16]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
+
+* notification integrations (create/edit/delete).
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:118]:
```suggestion
* Notification integrations (create/edit/delete)
```

File: modules/rbac-resource-definitions.adoc [hunk 17]
```diff
@@ -47,61 +65,61 @@ _(Local administrator only.)_
...
+//TODO: Add link to API tokens
+//TODO: Add link to AWS S3 integration
+//TODO: Add link to image registry integrations
 
-| Indicator
-| View process activity in deployments.
-| N/A
 
 | K8sRole
 | View roles for Kubernetes role-based access control in secured clusters.
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:125]:
```suggestion
| View roles for Kubernetes RBAC in secured clusters.
```

File: modules/rbac-resource-definitions.adoc [hunk 18]
```diff
@@ -115,10 +133,6 @@ _(Local administrator only.)_
 | View users and groups for Kubernetes role-based access control in secured clusters.
```
Comment (kcarmichael08) [inline modules/rbac-resource-definitions.adoc:133]:
```suggestion
| View users and groups for Kubernetes RBAC in secured clusters.
```

**Review by kcarmichael08 — COMMENTED** [review]
Left feedback based on our Style Guide that we use. Thank you for all of the info!

**Review by kcarmichael08 — COMMENTED** [review]
LGTM! If this is for 4.0+, we would merge to `rhacs-docs`, then cherry-pick to `rhacs-docs-4.0` if that can be reflected in the body of the pull request. (`rhacs-docs` doesn't get published directly - just the branches do - and 4.0 won't get published until the release goes out so it can be merged at any point.) @gaurav-nelson can you PTAL and merge if OK?

**Comment by gaurav-nelson** [comment]
Thank you @kcarmichael08 LGTM. I'll get this merged.

**issue:** @rhybrillou Can you combine your commits into a single commit? This helps us to keep our commit history clean and makes it easier to fix content bleeding into different branches. This is required by our repository guidelines.

**Pull request merged**